Makefile

```diff
@@ -54,24 +54,26 @@ build/robot-validate.jar: | build
 build/robot-rdfxml.jar: | build
 	curl -Lk -o $@ https://build.obolibrary.io/job/ontodev/job/robot/job/mireot-rdfxml/lastSuccessfulBuild/artifact/bin/robot.jar
 
-### CINECA Tasks
+### GECKO Tasks
 
-data/cineca.tsv:
```

Comment:
It's a minor thing, but I think that `data/cineca.tsv` should not be renamed to "gecko". @mcourtot?

Reply:
sounds right - it helps showing the provenance from cineca in that case. agree it is minor too.

Reply:
True. Let’s keep it.

Reply:
Sorry, I mixed up my replies! Yes let’s call this table cineca.

Reply:
Fixed